LVT/LVT.Services/InitiativePresenter.cs

```diff
@@ -0,0 +1,51 @@
+﻿using LVT.LVT.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LVT.LVT.Services
+{
+    public class InitiativePresenter : IVisualizer<Initiative>
+    {
+        //private IVisualizer<Epic> _epicpre;
+        //public InitiativePresenter(IVisualizer<Epic> epicpre)
+        //{
+        //    _epicpre = epicpre;
+        //}
```

Comment:
Delete this commented out code if it's not used. If it's useful describe why

Reply:
This was left in as a reminder for testing - something one of the colleagues here suggested doing when we were asking about how to use mocks.  Should be resolved as we continue testing.